darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -916,6 +916,167 @@ def fit_from_dataset(
         self._train(train_loader, val_loader)
         return self
 
+    @staticmethod
+    def setup_finetuning(
+        old_model_name: str,
+        new_model_name: str = None,
+        additional_epochs: int = 0,
```

Comment:
Do we need this parameter, given that `fit()` already accepts `epochs` ?
I would find it cleaner to rely exclusively on `fit()`'s parameter. If there's a problem with it, could we maybe fix it there (i.e. handle the trainer correctly in `fit()` to handle `epoch`)?

Reply:
There is an issue with the `fit()` parameter (#1495), I think that @rijkvandermeulen is already working on a fix. I will remove the `epochs` argument from this method and wait for the patch to be merged.